### ISSUE TITLE
Bug Report: Docker image not found

### ISSUE THREAD

**Issue by ajmeese7** [bonfire-networks/bonfire-app#108]
**Describe the bug**
When I run `make rel.run` I get the following error:

```
ERROR: manifest for bonfirenetworks/bonfire:latest-classic not found: manifest unknown: manifest unknown
```

If I try to build my own image with `make rel.build`, I get the following error:

```
cp: cannot overwrite non-directory './data/current_flavour/classic' with directory 'flavours/classic'
make: *** [Makefile:322: rel.config.prepare] Error 1
```

**Desktop (please complete the following information):**
 - OS: Raspbian GNU/Linux 11 (bullseye)
 - Kernel: Linux 5.10.63-v7l+
 - Version:  0.1.0-beta.71 - 4088f45b

**Comment by mayel** [comment]
Hiya, thanks for the report! we haven't set up CI to automatically build an image for the classic flavour but will do so now.

**Comment by mayel** [comment]
Nice to see you trying it on Rasbian BTW, keen to know how it goes!

**Comment by ajmeese7** [comment]
I will be looking forwards to it! Do you suggest I try the Cooperation theme in the meantime?

**Comment by mayel** [comment]
You can't because that image is only built for amd64... Hold on, the CI is running now: https://github.com/bonfire-networks/bonfire-app/runs/4495910618?check_suite_focus=true

**Comment by mayel** [comment]
There we go, looks like an image for armv7 is ready: https://hub.docker.com/repository/docker/bonfirenetworks/bonfire/tags

It did take 1h+ instead of 5-10 min, so I might split it into another CI task other archs build quicker rather than in parallel...

**Comment by ajmeese7** [comment]
That's a great idea, I fully intend to continue using this project on my Pi so I would appreciate that.

Thanks for the fix, it worked like a charm! Closing this issue out now 🎉
